pyface/tasks/tests/test_enaml_dock_pane.py

```diff
@@ -1,14 +1,19 @@
 from traits.testing.unittest_tools import unittest
 from traits.etsconfig.api import ETSConfig
 
+# Skip tests if Enaml is not installed or we're using the wx backend.
+SKIP_REASON = None
 if ETSConfig.toolkit not in ['', 'qt4']:
-    raise unittest.SkipTest("TestEnamlDockPane: Enaml does not support WX")
-
+    SKIP_REASON = "TestEnamlTaskPane: Enaml does not support WX"
 try:
     from enaml.widgets.api import Label
     from traits_enaml.testing.gui_test_assistant import GuiTestAssistant
 except ImportError:
-    raise unittest.SkipTest("Enaml not installed")
+    SKIP_REASON = "Enaml not installed"
```

Comment:
Hmm. Not quite right: if we're using the wx backend, we'll likely end up overwriting the original `SKIP_REASON` here. Fixing ...